slack_bolt/context/complete/complete.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, Dict, Optional
+
+from slack_sdk import WebClient
+from slack_sdk.web import SlackResponse
+
+
+class Complete:
+    client: WebClient
+    function_execution_id: Optional[str]
+
+    def __init__(
+        self,
+        client: WebClient,
+        function_execution_id: Optional[str],
+    ):
+        self.client = client
+        self.function_execution_id = function_execution_id
+
+    def __call__(self, outputs: Dict[str, Any] = {}) -> SlackResponse:
```

Comment:
If you need a default value here, you should not use an empty dict value for argument default. See https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python for more details
```suggestion
    def __call__(self, outputs: Optional[Dict[str, Any]] = None) -> SlackResponse:
```
And the following lines of code should be updated accordingly.

If the default value is not necessary, you can simply remove it.

Reply:
Wow TIL, did not think the default value would be assigned to the function itself 😮